app/src/main/java/com/example/lhi06/residue/FindAlbumActivity.java

```diff
@@ -58,6 +59,9 @@ public void onFailure(final Call<AlbumSearchResponse> call, final Throwable t) {
 
         } catch (IOException e) {
             e.printStackTrace();
+        } catch (NullPointerException e) {
```

Comment:
It's never the right thing to do to catch a NullPointerException as, if you end up having to do this, it means there is a bug in your code and all all your are doing is hiding the bug (the only possible exception to this would be if you are using a third party lib which has a bug and there id no alternative lib you could use)